index.ts

```diff
@@ -129,10 +171,9 @@ module.exports.fromLatlng = module.exports.fromLeaflet = function fromLatlng (lo
 
  var position = lonlat.fromPoint({ x: 12, y: 34 })   // { lon: 12, lat: 34 }
  */
-function fromPoint (point) {
-  return floatize({lon: point.x, lat: point.y})
+export const fromPoint = function (point: Point): LonLatOutput {
```

Comment:
I personally prefer this syntax for functions (also for all funcs in this file).

```suggestion
export function fromPoint (point: Point): LonLatOutput {
```

Reply:
This is my preferred style also.

Reply:
I did this weird double one so that we could have the space between the function name and the parameters (since prettier squishes them otherwise) but I do agree shorter is better. Changed in 1711f39